src/mainboard/protectli/vault_jsl/Makefile.inc

```diff
@@ -12,5 +12,6 @@ romstage-y += die.c
 ramstage-y += die.c
 smm-y += die.c
 
-SPD_SOURCES = MT53E512M32D4NQ-053 # 2GBx4
+#SPD_SOURCES = MT53E512M32D4NQ-053 # 2GBx4
```

Comment:
@miczyg1 Does it make sense to leave some more info in the comment?

For example:

```
-#SPD_SOURCES = MT53E512M32D4NQ-053 # 2GBx4
-SPD_SOURCES = MT53D512M64D4RQ-046 # 2GBx4
+#SPD_SOURCES = MT53E512M32D4NQ-053 # 2GBx4 (rc3 prototype unit)
+SPD_SOURCES = MT53D512M64D4RQ-046 # 2GBx4 (v1.0 production unit
```

Reply:
In case we would need some backwards compatibility later or would like to come back to this, I have left the old SPD commented.

Reply:
https://github.com/Dasharo/coreboot/pull/503/files#r1599876320